docker-jans-fido2/scripts/upgrade.py

```diff
@@ -31,7 +31,7 @@ def _transform_fido2_dynamic_config(conf):
         ("oldU2fMigrationEnabled", True),
         ("metadataUrlsProvider", ""),
         ("errorReasonEnabled", False),
-        ("skipDownloadMdsEnabled", False),
+        ("disableMetadataService", False),
```

Comment:
Revert the changes for this line, as it will be taken care after this PR merged and artifact is rebuilt. 

Ref:https://github.com/JanssenProject/jans/issues/9146

Reply:
You can revert any changes to docker-* subrepo. CN team will sync the changes after your PR merged to main branch.